var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -40,21 +40,18 @@ class Jdk(Package):
     # commandline options.
     #
     # See http://stackoverflow.com/questions/10268583/how-to-automate-download-and-installation-of-java-jdk-on-linux
-    curl_options = [
-        '-j',  # junk cookies
-        '-H',  # specify required License Agreement cookie
-        'Cookie: oraclelicense=accept-securebackup-cookie']
+    headers = ['Cookie: oraclelicense=accept-securebackup-cookie']
```

Comment:
Does this still fetch after this change? I figured the License Agreement flag is required.

Reply:
It works for me. Which flag do you mean? The cookie header still gets set like before the change.